orchagent/main.cpp

```diff
@@ -143,14 +160,12 @@ int main(int argc, char **argv)
     int opt;
     sai_status_t status;
 
-    bool disableRecord = false;
-
     while ((opt = getopt(argc, argv, "m:hR")) != -1)
     {
         switch (opt)
         {
         case 'R':
-            disableRecord = true;
+            gDisableRecord = true;
```

Comment:
we should have separate flag to enable this? 0 disable, 1 swss, 2, sairedis, 3 both.